code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -174,8 +174,8 @@ Borg Hypospray
 Borg Shaker
 */
 /obj/item/reagent_containers/borghypo/borgshaker
-	name = "cyborg shaker"
-	desc = "An advanced drink synthesizer and mixer."
+	name = "cyborg soda shaker"
```

Comment:
At this point, maybe just create separate soda and alcoholic shaker, and leave borgshaker as a parent class for them. Keep the old name, move name/desc and reagents to the soda shaker.